packages/node/__tests__/index.test.ts

```diff
@@ -150,6 +138,43 @@ describe('#metrics', () => {
       });
   });
 
+  it('express should log the full request url with nested express apps', () => {
+    const apiMock = getReadMeApiMock(1);
+    const mock = nock(config.host, {
+      reqheaders: {
+        'Content-Type': 'application/json',
+        'User-Agent': `${pkg.name}/${pkg.version}`,
+      },
+    })
+      .post('/v1/request', ([body]) => {
+        expect(body.group).toStrictEqual(outgoingGroup);
+        expect(body.request.log.entries[0].request.url).toContain('/test/nested');
+        return true;
+      })
+      .basicAuth({ user: apiKey })
+      .reply(200);
+
+    const app = express();
+    const appNest = express();
+
+    appNest.use(expressMiddleware(apiKey, () => incomingGroup));
+    appNest.get('/nested', (req, res) => {
+      expect(req.url).toBe('/nested');
```

Comment:
```suggestion
      // We're asserting `req.url` to be `/nested` here because the way that Express does contextual route loading
      // `req.url` won't include the `/test`. The `/test` is only added later internally in Express with `req.originalUrl`.
      expect(req.url).toBe('/nested');
```